packages/stylus/plugin/compile-stylus.js

```diff
@@ -89,16 +90,21 @@ class StylusCompiler extends MultiFileCachingCompiler {
     }
 
     const importer = {
-      find(importPath, paths) {
+      find(importPath, paths, filename) {
```

Comment:
Ah, this is my habit. By looking at Stylus code, I realized it is being passed in. I like to then list all arguments on the callee said as well, because maybe in the future somebody will want to fix something here and spend hours doing a workaround, when they would just have this value in the argument. I can remove it, but to me it is a very useful practice.

Reply:
Okay, thanks @mitar - we can leave it.